src/SDKs/Cdn/Management.Cdn/Microsoft.Azure.Management.Cdn.csproj

```diff
@@ -5,7 +5,7 @@
 
 	<PropertyGroup>
 		<Description>Microsoft Azure CDN Management Library</Description>
-		<Version>4.0.1-preview</Version>
+		<Version>4.0.2-preview</Version>
 		<AssemblyName>Microsoft.Azure.Management.Cdn</AssemblyName>    
 		<PackageId>Microsoft.Azure.Management.Cdn</PackageId>
 		<PackageTags>Microsoft Azure CDN management;CDN;Cdn management;</PackageTags>
```

Comment:
Please update the `PackageReleaseNotes` here

Reply:
Release Notes updated